cdc/owner/changefeed.go

```diff
@@ -360,15 +365,17 @@ func (c *changefeed) tick(ctx cdcContext.Context, captures map[model.CaptureID]*
 		return nil
 	}
 
-	// If the owner is just initialized, barrierTs can be `checkpoint-1`.
+	// If the owner is just initialized, GlobalBarrierTs can be `checkpointTs-1`.
 	// In such case the `newResolvedTs` and `newCheckpointTs` may be larger
-	// than the barrierTs, but it shouldn't be, so we need to handle it here.
-	if newResolvedTs > barrierTs {
-		newResolvedTs = barrierTs
+	// than the GlobalBarrierTs, but it shouldn't be, so we need to handle it here.
+	if newResolvedTs > barrier.GlobalBarrierTs {
```

Comment:
we can safely remove the logic here, since the global resolvedTs has been decoupled  decoupled from the globalBarrierTs. Ref:
![image](https://user-images.githubusercontent.com/61726649/225537029-29be2fdb-0e91-48f8-ab78-2634f8730061.png)